tools/daemon/Makefile.am

```diff
@@ -1,13 +1,13 @@
 sbin_PROGRAMS = xliod
 
-xliod_LDADD =
+xliod_LDADD = $(DOCA_LIBS)
 
 xliod_CPPFLAGS = \
 	-I$(top_srcdir)/src \
 	-I$(top_srcdir)/src/core
 
 xliod_LDFLAGS  = 
-xliod_CFLAGS = 
+xliod_CFLAGS = -Wno-deprecated-declarations
```

Comment:
why

Reply:
we use experimental doca apis that have deprecation warning